### PR TITLE
526 Chapter 3 Content

### PR DESCRIPTION
**Pull request by cvalarida**
### Supporting evidence orientation
image

### Evidence type selection
image

### VA medical records intro
image

### VA facilities
image

### Private medical records intro
image

### Private medical records choice
image
Not sure how I like the address formatting in there...Do you have opinions, @goldenmeanie?

image

### Private medical records upload
image

### Additional evidence
image

### CODE REVIEW

File: src/applications/disability-benefits/526EZ/helpers.jsx [hunk 1]
```diff
@@ -15,6 +15,8 @@ const siblings = ['treatments', 'privateRecordReleases', 'privateRecords', 'addi
 import { PREFILL_STATUSES } from '../../common/schemaform/save-in-progress/actions';
 import { DateWidget } from '../../common/schemaform/review/widgets';
 
+const vaForm4142URL = 'https://www.vba.va.gov/pubs/forms/VBA-21-4142-ARE.pdf';
```
Comment (markgreenburg) [inline src/applications/disability-benefits/526EZ/helpers.jsx]: Good call 👍

File: src/applications/disability-benefits/526EZ/helpers.jsx [hunk 2]
```diff
@@ -118,7 +120,7 @@ export const disabilityNameTitle = ({ formData }) => {
 
 export const facilityDescription = ({ formData }) => {
   return (
-    <p>Tell us about facilities where VA treated you for {getDiagnosticCodeName(formData.diagnosticCode)}, <strong>after you got your disability rating</strong>.</p>
+    <p>Please tell us where VA treated you for {getDiagnosticCodeName(formData.diagnosticCode)} <strong>after you got your disability rating</strong>.</p>
```
Comment (markgreenburg) [inline src/applications/disability-benefits/526EZ/helpers.jsx]: [Not something to address in this PR]
`getDiagnosticCodeName` returns 'Unknown Condition' in the case that a mapping isn't available for the input diagnostic code. Maybe 'condition' would be more appropriate as default text. I realize this is an edge-case either way.

File: src/applications/disability-benefits/526EZ/helpers.jsx [hunk 3]
```diff
@@ -173,28 +175,28 @@ export const privateRecordsChoiceHelp = (
...
-      <a href="https://www.vba.va.gov/pubs/forms/VBA-21-4142-ARE.pdf" target="_blank">
+      <a href={vaForm4142URL} target="_blank">
         Download VA Form 21-4142
       </a>.
     </p>
   </AdditionalInfo>
 );
 
-const firstOrNextString = (evidenceTypes) => (evidenceTypes['view:vaMedicalRecords'] ? 'next' : 'first');
+const firstOrNextString = (evidenceTypes) => (evidenceTypes['view:vaMedicalRecords'] ? 'Now' : 'First,');
```
Comment (markgreenburg) [inline src/applications/disability-benefits/526EZ/helpers.jsx]: You could rename this `const` to `firstOrNowString` if you wanted to keep the semantics. I don't have a strong opinion either way.

File: src/applications/disability-benefits/526EZ/helpers.jsx [hunk 4]
```diff
@@ -283,7 +295,13 @@ export const recordReleaseWarning = (
 export const documentDescription = () => {
   return (
     <div>
-      <p>File upload guidelines:</p>
+      <p>
+        You can upload your document in a pdf, .jpeg, or .png file format. You’ll
```
Comment (markgreenburg) [inline src/applications/disability-benefits/526EZ/helpers.jsx]: This isn't the complete set of accepted file formats, though I don't know if we want to surface all of the accepted formats in this content snippet.

File: src/applications/disability-benefits/526EZ/helpers.jsx [hunk 5]
```diff
@@ -240,14 +242,24 @@ export function validateAddress(errors, formData) {
 export const download4142Notice = (
   <div className="usa-alert usa-alert-warning no-background-image">
     <p>
-      Since your doctor has your private medical records, you’ll need to fill
+      Since your doctor has your private medical records, you'll need to fill
       out an Authorization to Disclose Information to the VA (VA Form 21-4142) so
-      we can request your records. You’ll need to fill out a form for each doctor.
```
Comment (markgreenburg) [inline src/applications/disability-benefits/526EZ/helpers.jsx]: I thought we wanted to include the part about filling out one form per doctor? Just checking.

File: src/applications/disability-benefits/526EZ/helpers.jsx [hunk 6]
```diff
@@ -240,14 +242,24 @@ export function validateAddress(errors, formData) {
...
     </p>
     <p>
-      <a href="https://www.vba.va.gov/pubs/forms/VBA-21-4142-ARE.pdf" target="_blank">
+      <a href={vaForm4142URL} target="_blank">
         Download VA Form 21-4142
       </a>.
+      <p>
+        Please print the form, fill it out, and send it to:
+      </p>
+      <p className="va-address-block">
```
Comment (markgreenburg) [inline src/applications/disability-benefits/526EZ/helpers.jsx]: I believe this address is used in at least two places:
1. On this page's content
2. On the confirmation page's content

Would it be worth pulling the address `jsx` out into its own variable?

File: src/applications/disability-benefits/526EZ/helpers.jsx [hunk 7]
```diff
@@ -296,8 +314,14 @@ export const documentDescription = () => {
 export const additionalDocumentDescription = () => {
   return (
     <div>
-      <p>If you have other evidence, like lay or buddy statements, that you would
-        like to submit, you can upload them here.</p>
+      <p>
+        If you have other evidence, such as a lay or buddy statement to turn in,
+        you can upload them here. You can upload your document in a pdf, .jpeg, or
+        .png file format. You’ll first need to scan a copy of your document onto
```
Comment (markgreenburg) [inline src/applications/disability-benefits/526EZ/helpers.jsx]: Reminder to double-check file formats.

**Review by markgreenburg — APPROVED** [review]
Had a few non-blocking questions and comments, but looks good!

**Review by cehsu — APPROVED** [review]
LGTM pending Peggy's approval.

File: src/applications/disability-benefits/526EZ/helpers.jsx (hunk 5 above)
Comment (cvalarida) [reply src/applications/disability-benefits/526EZ/helpers.jsx]: Ohhh, I added it back to the wrong place. Good catch.

**Pull request merged**